lib/fizzy/instantiate.hpp

```diff
@@ -87,13 +87,20 @@ struct Instance
 };
 
 // Instantiate a module.
-std::unique_ptr<Instance> instantiate(Module module,
+std::unique_ptr<Instance> instantiate(std::unique_ptr<Module> module,
     std::vector<ExternalFunction> imported_functions = {},
     std::vector<ExternalTable> imported_tables = {},
     std::vector<ExternalMemory> imported_memories = {},
     std::vector<ExternalGlobal> imported_globals = {},
     uint32_t memory_pages_limit = DefaultMemoryPagesLimit);
 
+// Instantiate a module.
+std::unique_ptr<Instance> instantiate(Module _module,
```

Comment:
It is used only in tests and once in spectests, probably better to move to `test/`